.github/workflows/ci-conda-known-test-failures.json

```diff
@@ -1,12 +1,18 @@
 {
+    "doc.en.constructions.calculus": {
+        "failed": "unreported random failures in plotting"
+    },
     "sage_setup.clean": {
-        "failed": true
+        "failed": "_find_stale_files finds some stale files under sage/tests when executed under conda"
+    },
+    "sage.algebras.fusion_rings.fusion_ring": {
+        "failed": "unreported random timeouts"
     },
     "sage.combinat.cluster_algebra_quiver.quiver": {
-        "failed": true
+        "failed": "failure seen in https://github.com/sagemath/sage/actions/runs/6836592771/job/18591690058#step:11:10059"
```

Comment:
These comments are pretty useless as the logs are deleted quite quickly (90 days?)

Reply:
Yes, 90 days. Enough time for people to investigate and open Issues / PRs.

Not helpful to demand that it's done here.

Reply:
You missed step 2 of your own procedure (https://github.com/sagemath/sage/pull/36678#issuecomment-1837386861):
>
    Test failure is noticed
    You record it in a GitHub Issue. There you include the necessary information so that it does not take 2 hours to reconstruct it.
    One adds stuff to the exclusion list to eliminate noise in the tests. It's not for humans to read, and definitely not for humans to use 2 hours of guessing time to reconstruct the failure.
    The exclusion list is maintained like everything else in Sage -- by PRs, not by specific people in charge.